docs/requirements.txt

```diff
@@ -1,5 +1,5 @@
 sphinx==2.4.4
-sphinx-gallery
+sphinx-gallery>=0.9.0
 matplotlib
 numpy
```

Comment:
Actually, we can avoid NumPy dependency. `matplotlib` covers it?

Reply:
You're right, but it doesn't hurt to be explicit for those things IMHO.
Who knows: we might end up not using matplotlib for showing images, or mpl may even not depend on numpy at some point (quite unlikely I agree)

Reply:
I think the best practice is to we avoid transitive dependencies and define explicitly. With a quick look on the documentation folder, I don't see a direct import on `numpy` (only some references on the config) but if it is a necessary dependency I would leave it explicitly defined for the reasons that Nicolas explained.

Reply:
We have an explicit dep on numpy in `gallery/plot_transforms.py` which isn't in the doc folder, but it's still part of the doc building